### PR TITLE
Improve documentation of CYPRESS_DOWNLOAD_PATH_TEMPLATE

### LINKED ISSUE #4836: CYPRESS_DOWNLOAD_PATH_TEMPLATE documentation is incomplete
### Subject

References

### Description

The documentation for CYPRESS_DOWNLOAD_PATH_TEMPLATE (located at https://docs.cypress.io/guides/references/advanced-installation#Download-URLs) currently lacks:
- Information on what exactly the template variables are replaced with
- Examples of downloading from a different host/server
- Information on usage in `.npmrc`
- Information on which Cypress versions can be installed with it

### PR DESCRIPTION
**Pull request by JessefSpecialisterren**
Gives `CYPRESS_DOWNLOAD_PATH_TEMPLATE` its own section documenting it in a lot more detail, and also fixes a typo 🙂

Closes #4836

### CODE REVIEW

File: content/guides/references/advanced-installation.md [hunk 1]
```diff
@@ -180,6 +170,50 @@ CYPRESS_DOWNLOAD_MIRROR="https://www.example.com" cypress install
 Cypress will then attempt to download a binary with this format:
 `https://www.example.com/desktop/:version?platform=p`
 
+## Download path template
+
+Starting with Cypress 9.3.0, you can use the `CYPRESS_DOWNLOAD_PATH_TEMPLATE`
+environment variable to make Cypress download the binary from a custom URL
```
Comment (emilyrohrbough) [inline content/guides/references/advanced-installation.md:176]:
```suggestion
environment variable to download the Cypress binary from a custom URL
```

File: content/guides/references/advanced-installation.md [hunk 2]
```diff
@@ -180,6 +170,50 @@ CYPRESS_DOWNLOAD_MIRROR="https://www.example.com" cypress install
...
+that's generated based on version, platform and architecture.
```
Comment (emilyrohrbough) [inline content/guides/references/advanced-installation.md:177]:
```suggestion
that's generated based on endpoint, version, platform and architecture.
```

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@JessefSpecialisterren Thank you so much for this contribution! This is way clearer!

**Comment by JessefSpecialisterren** [comment]
@emilyrohrbough Glad to hear it, and thanks for approving the PR 🙂! I don't know the procedure from here, do you expect me to do something with the changes you suggested? If I push additional commits, won't that dismiss your approval of the PR?

**Pull request merged**